specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/search.json

```diff
@@ -616,6 +616,57 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Search/searchServices/{searchServiceName}/privateLinkResources": {
+      "get": {
+        "tags": [
+          "PrivateLinkResources"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
>null [](start = 26, length = 4)

"nextLink"

Reply:
This particular one I left as is as we chose not to add it in the backend.  I presume this is because this return item isn't configurable by the user, and will only ever be a fixed list that we send, and we don't expect it to get so long that paging would be necessary/useful.